### ISSUE TITLE
twisted.python.deprecate module docstring refers to non-existence deprecatedModuleAttributes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4235]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4235|
|Type|defect|
|Created|2010-01-27 02:33:09Z|

`deprecatedModuleAttributes` was renamed to `deprecatedModuleAttribute`.  A few places in the docstring were missed, though.

Attachments:

* [wrong-name.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/633/63366435425b24cdba50313b666342a112aaefc1/8936b7d078fac41d352dfb52194cd9ab86643b79.patch) (1014 bytes) - added by exarkun on 2010-01-27 13:52:39Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4235 4235
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__easy documentation, easy
time__1264559589000000 1264559589000000
changetime__1264609438000000 1264609438000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_2"></a>|@khorn set owner to @exarkun|
|-|-|

Looks good to me.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [28075]) Apply wrong-name.patch fixing wrong names in module docstring

Author: exarkun
Reviewer: khorn
Fixes: [#4235](https://github.com/twisted/twisted/issues/4235)

Fix the references to `deprecatedModuleAttributes` in the module
docstring for `twisted.python.deprecate`.  The function is actually
named `deprecatedModuleAttribute`.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
